app/models/activity.rb

```diff
@@ -4,4 +4,6 @@ class Activity < ActiveRecord::Base
               in: %w(inside outside),
               message: "is not either 'inside' or 'outside'"
             }
+  validates_presence_of :lng
+  validates_presence_of :lat
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.